tools/toolchains/glibc/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+SRC_DIR=/tmp/glibc/glibc
+BUILD_DIR=/tmp/glibc/glibc_build
+INSTALL_DIR=/usr/local/occlum-glibc
+
+# Exit if any command fails
+set -e
+
+# Clean previous build and installation if any
+rm -rf ${SRC_DIR}
+rm -rf ${BUILD_DIR}
+rm -rf ${INSTALL_DIR}
+
+mkdir -p ${SRC_DIR}
+cd ${SRC_DIR}
+# Download glibc
+git clone -b occlum-glibc-2.27 https://github.com/occlum/glibc .
+
+mkdir -p ${BUILD_DIR}
+cd ${BUILD_DIR}
+# Build and install glibc
+unset LD_LIBRARY_PATH
+CFLAGS="-O2 -Wp,-U_FORTIFY_SOURCE -fno-stack-protector -Wno-unused-value -g" ${SRC_DIR}/configure \
```

Comment:
Is it possible to enable stack-protector and enable tests?

Reply:
Add new commit ”Initialize a randon number on stack for the stack protector“ to support stack protector